web/src/main/java/se/inera/intyg/webcert/web/service/facade/util/UtkastToCertificateConverterImpl.java

```diff
@@ -106,6 +111,10 @@ private Certificate convertToCertificate(Utkast certificate) {
                 certificateToReturn.getMetadata().getTypeVersion())
         );
 
+        if (webCertUserService.hasAuthenticationContext() && webCertUserService.getUser().getParameters() != null) {
```

Comment:
Just a suggestion for increased code readability. Encapsulate the getResponsibleHospName in a private method where you can include the null checking etc.